shell/composables/useCompactInput.ts

```diff
@@ -0,0 +1,21 @@
+import { computed } from 'vue';
+import type { ComputedRef } from 'vue';
```

Comment:
Is there a reason to keep these separate?

Reply:
It's note required, I tend to prefer to try and keep it clear when an import is only used for type annotations. I'm not opposed to merging the two imports.